src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Author.scala

```diff
@@ -46,7 +44,8 @@ case class Author(titles: Option[String] = None,
       case (true, true) => s"$name; ${ organization.getOrElse("") }"
```

Comment:
if you would match on the `Some`/`None` instances, rather than on `Boolean`s, this piece of code would:
* be much more readable
* not require `getOrElse`